generators/typescript-mcp/base/src/ast/ObjectLiteralNode.ts

```diff
@@ -0,0 +1,32 @@
+import { ts } from "@fern-api/typescript-ast";
+
+export interface ObjectLiteralField {
+    name: string;
+    value: ts.AstNode;
+}
+
+export declare namespace ObjectLiteralNode {
+    interface Args {
+        fields: ObjectLiteralField[];
+    }
+}
+
+// TODO: generalize and move into @fern-api/typescript-ast
```

Comment:
Sounds good - in the future, the `base` is used for generator-specific abstractions not tied to the AST. We can leave this as-is for now, but I could see these being in something like `generators/typescript-mcp/ast` in the future.